### PR TITLE
examples: Update attribute names and backend for synthetics notebook

### PR DESCRIPTION
**Pull request by EdCaunt**
Updated pyvista to use new attribute names and notebook backend in the synthetics notebook. Should hopefully fix the notebook CI failures.

### CODE REVIEW

File: examples/seismic/tutorials/14_creating_synthetics.ipynb [hunk 1]
```diff
@@ -48,7 +48,8 @@
      "name": "stdout",
```
Comment (georgebisbas) [inline examples/seismic/tutorials/14_creating_synthetics.ipynb:48]: <pre class="ql-syntax" spellcheck="false">Line #24.    try:
</pre><p>Should we have version check for all python packages?</p><p>Maybe you could also try to see whether newer pyvista or gempy versions can be available?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (georgebisbas) [inline examples/seismic/tutorials/14_creating_synthetics.ipynb:48]: <p>missing plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Comment (georgebisbas) [inline examples/seismic/tutorials/14_creating_synthetics.ipynb:48]: <p>missing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Comment (EdCaunt) [reply examples/seismic/tutorials/14_creating_synthetics.ipynb:48]: <p>Yeah, not sure why that's not showing up. It displays locally. Might be some interplay between the new vtk notebook backend and nbreview?</p>

Comment (EdCaunt) [reply examples/seismic/tutorials/14_creating_synthetics.ipynb:48]: Looks like it doesn't display on github's notebook reader either, but it gets rid of the warning locally

Comment (georgebisbas) [inline examples/seismic/tutorials/14_creating_synthetics.ipynb:48]: <p>you have your local env here printed in the warning</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

**Review by mloubout — APPROVED** [review]

**Pull request merged**